components/x-privacy-manager/src/radio-btn.jsx

```diff
@@ -0,0 +1,31 @@
+import { h } from '@financial-times/x-engine';
+
+import s from './radio-btn.scss';
+
+export function RadioBtn({ value, checked, onChange, children }) {
+	const id = `ccpa-${value}`;
+
+	return (
+		<div className={s.control}>
+			<input
+				className={s.input}
+				id={id}
+				type="radio"
+				name="consent"
+				value={value}
+				checked={checked}
+				onChange={onChange}
+			/>
+			<label htmlFor={id} className={s.label}>
+				<span className={s.label__text}>
+					{children}
+				</span>
+
+				<svg className={s.label__icon} viewBox="0 0 36 36">
```

Comment:
If this image is used for presentational purposes only, you should consider using `aria-hidden="true"` to ensure it does not impact the accessible name for the label.

You should also consider using `focusable="false"` to prevent Internet Explorer from adding this image to the tab order.

https://stackoverflow.com/questions/29393144/how-to-prevent-svg-elements-from-gaining-focus-with-tabs-in-ie11

Reply:
Nice catch & thank you for the link!